apps/api/package.json

```diff
@@ -42,12 +42,14 @@
     "@nestjs/swagger": "^7.3.1",
     "@nestjs/typeorm": "^10.0.2",
     "@types/bcrypt": "^5.0.2",
+    "aws-sdk": "^2.1654.0",
     "axios": "^1.7.2",
     "bcrypt": "^5.1.1",
     "bull": "^4.12.9",
     "bullmq": "^5.8.3",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.1",
+    "firebase-admin": "^12.2.0",
```

Comment:
Do we need this package for AWS SNS?

Reply:
This was added to check fcm will remove it

Reply:
removed in https://github.com/OsmosysSoftware/osmo-x/pull/269/commits/201e68ef2916657f9528af193ce588e661831389